protocol.go

```diff
@@ -74,6 +74,15 @@ func (p *Listener) Accept() (net.Conn, error) {
 		WithPolicy(proxyHeaderPolicy),
 		ValidateHeader(p.ValidateHeader),
 	)
+
+	// If the ReadHeaderTimeout for the listener is 0, set a default of 200ms
+	if p.ReadHeaderTimeout == 0 {
+		p.ReadHeaderTimeout = 200 * time.Millisecond
```

Comment:
I think it could be helpful to export the default as a var so consumers of this library can override it. WDYT?

Reply:
Not sure what the benefit of that is over just setting the timeout itself. Do you have a use case that should work in?

Reply:
Changed just for simplicity

Reply:
For instance, one can override the default during compile time, eg `go build -ldflags -X "github.com/pires/go-proxyproto.DefaultReadHeaderTimeout=1s"` which would then apply too all uses of the lib, regardless of listener/conn.